adagrams/game.py

```diff
@@ -1,11 +1,129 @@
+import random
+
+# Function to draw 10 random letters from the letter pool
+
 def draw_letters():
-    pass
+    LETTER_POOL = {
```

Comment:
Since this dictionary is never changed, it would be good to have it declared outside of the function so that it doesn't have to be recreated each time the function is called.